stop_compaction_test.py

```diff
@@ -172,7 +175,7 @@ def _upgrade_sstables_format(node: BaseNode):
         finally:
             self.node.running_nemesis = False
 
-    def stop_reshape_compaction(self):
+    def stop_reshape_compaction(self, reshape_on_boot: bool = True):
```

Comment:
I think it might be better to call this arg "use_nodetool" or something similar. It makes sense in the context of the test itself to differentiate between reshaping and reshaping o boot, but what is actually happening is this flag switches between triggering reshape with nodetool or the REST API.

Reply:
it's not about using REST API over nodetool, it's more about restarting scylla server - and this restart is more important than the way of triggering stop. You didn't convince me :)